src/main/java/com/everis/cars/jms/CreateCarMessageListener.java

```diff
@@ -0,0 +1,87 @@
+package com.everis.cars.jms;
+
+import javax.ejb.EJB;
+import javax.ejb.MessageDriven;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import javax.jms.TextMessage;
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+import javax.json.bind.JsonbConfig;
+
+import org.apache.log4j.Logger;
+
+import com.everis.cars.control.CarService;
+import com.everis.cars.entity.Car;
+
+/**
+ * {@link MessageListener} implementation to listen the cars queue and create,
+ * update or delete received cars from queue
+ * 
+ * @see javax.ejb.MessageDriven
+ */
+@MessageDriven(mappedName = "jms/CarsAppQueue")
+public class CreateCarMessageListener implements MessageListener {
```

Comment:
is there a unit test for this class?

Reply:
I refactorized the class and created the test